src/include/cart/api.h

```diff
@@ -2247,6 +2247,51 @@ crt_quiet_error(int err)
 	return err == -DER_GRPVER;
 }
 
+/**
+ * Initialize and enable default quotas for the context
+ *
+ * \param[in] crt_ctx          CaRT context
+ *
+ * \return                     DER_SUCCESS on success, negative value on
+ *                             failure.
+ */
+int crt_context_quotas_init(crt_context_t crt_ctx);
+
+/**
+ * Finalize and disables quotas for the context
+ *
+ * \param[in] crt_ctx          CaRT context
+ *
+ * \return                     DER_SUCCESS on success, negative value on
+ *                             failure.
+ */
+int crt_context_quotas_finalize(crt_context_t crt_ctx);
+
```

Comment:
why are init / finalize public APIs ?

Reply:
I am still going back and forth whether we want to have a runtime control to disable (and subsequently re-enable) quotas. I started with that idea, which is why these are public, but for now auto-enabling it on every context